tests/xtro-sharpie/macOS-CoreMedia.ignore

```diff
@@ -3,3 +3,11 @@
 !missing-pinvoke! CMAudioDeviceClockGetAudioDevice is not bound
 !missing-pinvoke! CMAudioDeviceClockSetAudioDeviceID is not bound
 !missing-pinvoke! CMAudioDeviceClockSetAudioDeviceUID is not bound
+
+# as per headers: API_AVAILABLE(ios(9.0))
```

Comment:
It's unclear wrt docs 
https://developer.apple.com/documentation/coremedia/kcmsamplebufferlensstabilizationinfo_active?language=objc
but it's safer to exclude them until we're sure 👍